relay-server/src/actors/processor.rs

```diff
@@ -2259,6 +2291,12 @@ impl EnvelopeProcessorService {
     ) -> Result<(), ProcessingError> {
         let request_meta = state.managed_envelope.envelope().meta();
         let client_ipaddr = request_meta.client_addr().map(IpAddr::from);
+        let max_metric_name_and_unit_len = {
+            let max_mri_len = self.config.aggregator_config().max_name_length as isize;
+            // Length of mri without the parts 'name' and 'unit'.
+            let fixed_len = *FIXED_MEASUREMENT_LEN;
+            Some(max_mri_len - fixed_len)
```

Comment:
alternatively I can have it return a None value if max_mri_len is smaller than fixed_len. At least now i've changed it to an isize so it won't panic